cosmos-sdk/types/address_interface.go

```diff
@@ -0,0 +1,20 @@
+package types
```

Comment:
Inside juno we may need to derive an operator consensus address, in order to not depend on the sdk we use this builder that must be initalized when creating a new parser